src/osd/OSD.cc

```diff
@@ -33,6 +33,7 @@
 #include <sys/mount.h>
 #endif
 
+#include "common/numa.h"
```

Comment:
why are you reordering the `#include`?

Reply:
@tchaikov 
`OSD.cc` needs the `numa.h` include to actually make use of the (re)definitions I made before 
`#include "osd/PG.h"`
since that uses the `cpu_set_t`
Other alternative is to include `numa.h` in `PG.h` as well.
It is a matter of choice, and I considered this less invasive.

Reply:
> since that uses the `cpu_set_t`

please define "that", `git grep cpu_set_t | grep PG.h` returns nothing in my testbed.

Reply:
@tchaikov 
I put it back, and see what was the actual error, but it was due to missing some definitions.
Could also be from one of the includes IN `PG.h`

Reply:
@tchaikov 
I guess this one got fixed, when I loaded `numa.h` into `OSD.h`
Will take it out of the PR